src/Documenter.jl

```diff
@@ -312,6 +310,12 @@ deps = Deps.pip("pygments", "mkdocs")
 **`make`** is the function used to convert the markdown files to HTML. By default this just
 runs `mkdocs build` which populates the `target` directory.
 
+**`devbranch`** is the branch that "tracks" the in-development version of the  generated
+documentation. By default this value is set to `"master"`.
+
+**`devfolder`** the folder that in-development version of the docs will be deployed.
+Defaults to `"dev"`.
```

Comment:
Hmm.. this is now a breaking change, and will be somewhat chaotic for the users that their URLs suddenly change. Not sure how to handle it at the moment though.

Reply:
But then we also have to remove the `latest` directory if it exists so idk.